cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -3897,6 +4001,113 @@ func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeDegradedUnencrypted
 	c.Assert(filepath.Join(dirs.SnapBootstrapRunDir, fmt.Sprintf("%s-model-measured", s.sysLabel)), testutil.FilePresent)
 }
 
+func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeUnencryptedDataUnencryptedSaveHappy(c *C) {
```

Comment:
again if the comment is right s/Happy/Unhappy/?

Reply:
I've tweaked the comment, but the test is supposed to exercise a happy scenario where data is found to be unencrypted, and then proceeds to find save without any further calls to secboot.UnlockVolumeUsingKeyIfEncrypted.

Reply:
not really for  this but I wonder if we should log at least something if we there's a save of the wrong kind (unenc when expected encrypted and the other way around) that we ignore? maybe @anonymouse64 has thoughts on this

Reply:
I agree that kind of log message would be nice to have

Reply:
Missing or unencrypted save on an encrypted system will trigger and error which already gets logged. I guess we could additionally log something when we ignore encrypted save on an unencrypted system (it shouldn't be there but still), or fail to find ubuntu-save at all.